v1/src/test/java/com/google/cloud/teleport/templates/TextToBigQueryStreamLT.java

```diff
@@ -0,0 +1,266 @@
+          Field.of("score", StandardSQLTypeName.INT64),
+          Field.of("completed", StandardSQLTypeName.BOOL));
+
+  // 35,000,000 messages of the given schema make up approximately 10GB
+  private static final int NUM_MESSAGES_FOR_10GB = 35000000;
+  private static final long TIMEOUT_FOR_10_GB_TEST_MINUTES = 30;
+  private static final long TIMEOUT_FOR_1_HOUR_TEST_MINUTES = 60;
+
+  private static ArtifactClient gcsClient;
+  private static BigQueryResourceManager bigQueryResourceManager;
+
+  @Before
+  public void setup() throws IOException {
+    bigQueryResourceManager =
+        BigQueryResourceManager.builder(testName, project).setCredentials(CREDENTIALS).build();
+
+    Storage storageClient = createStorageClient(CREDENTIALS);
+
+    gcsClient = GcsArtifactClient.builder(storageClient, ARTIFACT_BUCKET, TEST_ROOT_DIR).build();
+  }
+
+  @After
+  public void teardown() {
+    ResourceManagerUtils.cleanResources(bigQueryResourceManager, gcsClient);
+  }
+
+  @Test
+  public void testBacklog10gb() throws IOException, ParseException, InterruptedException {
+    testBacklog(Function.identity());
+  }
+
+  @Test
+  public void testSteadyState1hr() throws IOException, ParseException, InterruptedException {
+    testSteadyState1hr(Function.identity());
+  }
+
+  private void testBacklog(
+      Function<PipelineLauncher.LaunchConfig.Builder, PipelineLauncher.LaunchConfig.Builder>
+          paramsAdder)
+      throws IOException, ParseException, InterruptedException {
+    // upload schema files and save path
+    String jsonPath =
+        getFullGcsPath(
+            ARTIFACT_BUCKET,
+            gcsClient
+                .uploadArtifact(
+                    "input/schema.json",
+                    Resources.getResource("TextIOToBigQueryTest/schema.json").getPath())
+                .name());
+
+    String udfPath =
+        getFullGcsPath(
+            ARTIFACT_BUCKET,
+            gcsClient
+                .uploadArtifact(
+                    "input/udf.js", Resources.getResource("TextIOToBigQueryTest/udf.js").getPath())
+                .name());
+
+    DataGenerator dataGenerator =
+        DataGenerator.builderWithSchemaTemplate(testName, "GAME_EVENT")
+            .setQPS("1000000")
+            .setMessagesLimit(String.valueOf(NUM_MESSAGES_FOR_10GB))
+            .setSinkType("GCS")
+            .setOutputDirectory(getTestMethodDirPath())
+            .setNumShards("20")
+            .setNumWorkers("50")
+            .setMaxNumWorkers("100")
+            .build();
+    // Executes the data generator
+    dataGenerator.execute(Duration.ofMinutes(TIMEOUT_FOR_10_GB_TEST_MINUTES));
+
+    /*
+     * This table will automatically expire 4000000 millis after creation if not cleaned up manually or by
+     * calling the {@link BigQueryResourceManager#cleanupAll()} method.
+     */
+    TableId table =
+        bigQueryResourceManager.createTable(testName, SCHEMA, System.currentTimeMillis() + 4000000);
```

Comment:
I think if you don't specify a expiry time, it expires in 1 hr? Can't we just do that?

Reply:
Yes. It expires in 1 hour. So, I have got the exception with cause: "com.google.cloud.bigquery.BigQueryException: Table apache-beam-testing.testSteadyState1hr_20230601_124309_193489.testSteadyState1hr does not have a schema."

Reply:
Yes. It's true. I have already fixed it.